README.md

```diff
@@ -43,28 +43,28 @@ Every helper exported by `react-select-event` takes a handle on the `react-selec
 Select one or more values in a react-select dropdown.
 
 ```jsx
-const { getByTestId, getByLabelText } = render(
-  <form data-testid="form">
+const { getByRole, getByLabelText } = render(
```

Comment:
Hope you are ok with this, you can find a form with `getByRole("form")` which I find pretty nice.